src/components/nav-links-sidebar.tsx

```diff
@@ -0,0 +1,82 @@
+'use client';
+
+import Link from 'next/link';
+import { usePathname } from 'next/navigation';
+
+import { Button, buttonVariants } from '@/components/ui/button';
+
+import { navLinks } from '@/config/nav-links';
+
+import { cn } from '@/lib/utils';
+
+import { useSidebarToggle } from '@/hooks/use-sidebar-toggle';
+import { useStore } from '@/hooks/use-store';
+
+import { LogOut } from 'lucide-react';
+
+export const NavLinksSidebar = () => {
```

Comment:
Vocês precisam definir um padrão pra forma como declaram componentes... Roberto tem usado function clássica, mas acredito que poderíamos manter como arrow function

Reply:
@roberto-juniorjp, this is a important topic that we need to discuss!

Reply:
@whyleonardo i actually like arrow functions, so i have no problems with that. Let's go Arrow then!